packages/components/psammead-caption/src/__snapshots__/index.test.jsx.snap

```diff
@@ -15,7 +15,6 @@ exports[`Caption should render with some offscreen text 1`] = `
 .c0 {
   font-size: 0.9375rem;
   line-height: 1.125rem;
-  color: #757575;
```

Comment:
This is highlighting that the Caption isn't pulling in the new `C_METAL` colour and the caption is actually losing it's grey colour. Quickly debugging this I realised that due to the mixup with `psammead-styles@1.0.1` not being in the `npm` registry that the package-lock for caption is wrong. 

To fix this checkout the branch, `cd packages/components/psammead-caption` then run `npm i` and the `package-lock,json` should update.

Reply:
Without package-lock update, black default browser colour (as per the snapshot output)
![image](https://user-images.githubusercontent.com/7791726/60595511-0f166e80-9d9f-11e9-9182-472859abdb29.png)

With new package-lock update, grey colour as expected. 
![image](https://user-images.githubusercontent.com/7791726/60595285-98797100-9d9e-11e9-83d4-b85a714b0d14.png)

Reply:
Confirmed this with @eleanombre, pushing the change now